plugins/channel_header_plug/channel_header_plug.jsx

```diff
@@ -120,7 +120,7 @@ export default class ChannelHeaderPlug extends React.PureComponent {
             <HeaderIconWrapper
                 buttonClass='channel-header__icon style--none'
                 iconComponent={plug.icon}
-                onClick={plug.action}
+                onClick={() => plug.action(this.props.channel, this.props.channelMember)}
```

Comment:
It would be nice to refactor this so that it doesn't use an arrow function since that'll cause HeaderIconWrapper to re-render each time this does